.gitpod.dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM gitpod/workspace-full-vnc:latest
+
+# Docker build does not rebuild an image when a base image is changed, increase this counter to trigger it.
+ENV TRIGGER_REBUILD 3
+
+# Install custom tools, runtime, etc.
+RUN sudo apt-get update \
+    # window manager
+    && sudo apt-get install -y jwm \
+    # electron
+    && sudo apt-get install -y libgtk-3-0 libnss3 libasound2 \
+    # native-keymap
+    && sudo apt-get install -y libx11-dev libxkbfile-dev \
+    && sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Do you have java installed on the container?

Reply:
Yes, it's present in the base image - here we just need to worry about the dependencies specific to this repo/project. The server seems to start fine on Gitpod (when it downloads), or at least the output is very similar if not identical to what I see on my physical machine,

Reply:
> Yes, it's present in the base image - here we just need to worry about the dependencies 

I should mention that the base Gitpod image is quite big and full-featured. It supports many languages, including Java.